ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -296,13 +274,10 @@ func (r *RayJobReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 			return ctrl.Result{RequeueAfter: RayJobDefaultRequeueDuration}, nil
 			// Job may takes long time to start and finish, let's just periodically requeue the job and check status.
 		}
-		if isJobPendingOrRunning(jobInfo.JobStatus) {
```

Comment:
This is checked by L303.

Reply:
I decide not to update this in this PR because this requires some changes in the dashboard HTTP client and some tests.